ml-agents/mlagents/trainers/curriculum.py

```diff
@@ -117,3 +111,27 @@ def get_config(self, lesson=None):
         for key in parameters:
             config[key] = parameters[key][lesson]
         return config
+
+    @staticmethod
+    def load_curriculum_file(location):
+        try:
+            with open(location) as data_file:
+                return Curriculum._load_curriculum(data_file)
+        except IOError:
+            raise CurriculumLoadingError(
+                "The file {0} could not be found.".format(location)
+            )
+        except UnicodeDecodeError:
```

Comment:
This should probably be handled in _load_curriculum instead, but I couldn't reproduce it to confirm :/